src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/client/PpudAutomationApiClientTest.kt

```diff
@@ -148,6 +149,29 @@ class PpudAutomationApiClientTest : IntegrationTestBase() {
     // no exception
   }
 
+  @Test
+  fun `update offence in ppud`() {
+    // given
+    val offenderId = "123"
+    val sentenceId = "456"
+    val id = "12345678"
```

Comment:
id will be superfluous when removed from ppudAutomationUpdateOffenceApiMatchResponse